shared/message/platform/webots/messages.proto

```diff
@@ -1,6 +1,6 @@
 // MIT License
 //
-// Copyright (c) 2021 NUbots
+// Copyright (c) 2025 NUbots
```

Comment:
I think we keep this as the year it was on creation? then it's implictily [creation year - present year]